toolbox

```diff
@@ -1778,14 +1812,17 @@ fi
 echo "$base_toolbox_command: TOOLBOX_PATH is $TOOLBOX_PATH" >&3
 
 if [ "$1" = "" ]; then
-    echo "$base_toolbox_command: missing command" >&2
-    echo >&2
-    echo "These are some common commands:" >&2
```

Comment:
On second thoughts, it would be better to retain this super abbreviated list. We had deliberately [added it](https://github.com/debarshiray/toolbox/commit/3afb2bc4c74db705af2e78bd92ee244597193970) to provider a softer landing ground than the full reference documentation for absolute beginners.

In some sense you can think of it is a self-documenting error message than documentation. Just like some of the other (error) messages that suggest possible solutions or next steps.